File: semester I/2310817310002_Jovan/C/Soal4.c
#include <stdio.h>

void main() {
    int VERTIKAL, HORIZONTAL;

    // VERTIKAL = Kolom
    // HORIZONTAL = Baris

    printf("Jumlah Kolom: ");
    scanf("%d", &VERTIKAL);
    printf("Jumlah Baris: ");
    scanf("%d", &HORIZONTAL);

    for (int i = 0; i < VERTIKAL; i++) {
        printf(" _ ");
    }

    printf("\n");

    for (int i = 0; i < HORIZONTAL; i++) {
        for (int j = 0; j < VERTIKAL; j++) {
            printf("|_|");
        }
        printf("\n");}
}

File: semester I/2310817310002_Jovan/Python/Soal4.py
VERTIKAL = int(input("Jumlah Kolom: "))
HORIZONTAL = int(input("Jumlah Baris: "))

for i in range(VERTIKAL):
    print(" _ ", end="")

print()

for i in range(HORIZONTAL):
    for j in range(VERTIKAL):
        print("|_|", end="")
        
    print()

